client/test/app/hearings/components/VirtualHearings/Timezone.test.js

```diff
@@ -130,9 +130,9 @@ describe('Timezone', () => {
   });
 
   test('Dropdown displays correct times based on props time and roTimezone', () => {
-    const time = HEARING_TIME_OPTIONS[0].value;
+    const time = "08:15 am";//HEARING_TIME_OPTIONS[0].value;
```

Comment:
```suggestion
    const time = "08:15 am";
```

Should `//HEARING_TIME_OPTIONS[0].value;` be kept instead?